src/sbt-test/docker/override-commands/build.sbt

```diff
@@ -9,7 +9,7 @@ version := "0.1.0"
 maintainer := "Gary Coady <gary@lyranthe.org>"
 
 dockerCommands := Seq(
-  Cmd("FROM", "dockerfile/java:latest"),
+  Cmd("FROM", "dockerfile/openjdk:latest"),
```

Comment:
Ah, sorry. I saw that afterwards. We need to remove the `dockerfile/`. I will do so and apply it to master and 1.1.x